oktoberfest/ce_calibration.py

```diff
@@ -128,17 +128,26 @@ def merge_mzml_and_msms(self, df_search: pd.DataFrame):
 
     def get_mzml_path(self) -> Path:
         """Get path to mzml file."""
-        mzml_path = self.out_path / "mzML"
+        if self.config.spectra_type == "mzml":
+            mzml_path = self.config.spectra
+        else:
+            mzml_path = self.out_path / "data"
         mzml_path.mkdir(exist_ok=True)
         return mzml_path / self.raw_path.with_suffix(".mzML").name
 
     def get_hdf5_path(self) -> Path:
         """Get path to hdf5 file."""
-        return self.out_path / "mzML" / self.raw_path.with_suffix(".mzML.hdf5").name
+        if self.config.spectra_type == "mzml":
+            return self.config.spectra / self.raw_path.with_suffix(".mzML.hdf5").name
```

Comment:
no, the hdf5 files should not go into the spectra folder! They are part of oktoberfest's output, i.e. have to go to out_path / data

Reply:
i.e. leave it as it was before but chenge mzml folder name to data